requirements-travis.txt

```diff
@@ -1,4 +1,4 @@
-pytest >= 3.2.1
+pytest==3.2.1
```

Comment:
This was a fix for errors observed in Travis caused by seemingly incompatible versions of `py` and `pytest`.

Reply:
Is it one of those bugs that appears on Travis but you can't replicate locally?

Reply:
It was [this one](https://travis-ci.org/hylang/hy/jobs/416043975).  I think I was able to replicate it locally, but it must've been caused by something in the old PR, because I can't now.

Reply:
You know, I think I saw that once, but I couldn't reproduce it. I say let Travis use the new version for now and let's try to fix that issue when we get the opportunity.